.github/workflows/release-sprint.yml

```diff
@@ -53,94 +54,234 @@ jobs:
-          --milestone ${{ steps.var.outputs.milestone }} \
-          --pre-note ./assets/release-sprint/pre-note.md \
-          --post-note ./assets/release-sprint/post-note.md \
-          --note-contributors innobead \
-          --note-section-disable \
-          --since-days 14 \
-          --exclude-labels "kind/test" \
-          --exclude-labels "area/infra" \
-          --exclude-labels "area/ci" \
-          --exclude-labels "area/qa" \
-          --exclude-labels "wontfix" \
-          --exclude-labels "duplicated" \
-          --exclude-labels "invalid" \
-          --exclude-labels "release/task" \
-          --pre-hook ./scripts/check-images-ready.sh \
-          --pre-hook-args "longhornio/longhorn-manager:${{ steps.var.outputs.tag }}" \
-          --pre-hook-args "longhornio/longhorn-engine:${{ steps.var.outputs.tag }}" \
-          --pre-hook-args "longhornio/longhorn-instance-manager:${{ steps.var.outputs.tag }}" \
-          --pre-hook-args "longhornio/longhorn-share-manager:${{ steps.var.outputs.tag }}" \
-          --pre-hook-args "longhornio/backing-image-manager:${{ steps.var.outputs.tag }}" \
-          --pre-hook-args "longhornio/longhorn-ui:${{ steps.var.outputs.tag }}" \
-          --pre-hook-args "longhornio/longhorn-cli:${{ steps.var.outputs.tag }}" \
-          --artifacts ./longhorn.yaml \
-          --artifacts ./longhorn-images.txt \
-          --artifacts ./charts.tar.gz \
-          --artifacts ./longhorn-images-sbom.tar.gz \
-          --pre-release \
+        common_args=(
+          --owner longhorn
+          --repo longhorn
+          --branch "${{ steps.var.outputs.branch }}"
+          --tag "${{ steps.var.outputs.tag }}"
+          --milestone "${{ steps.var.outputs.milestone }}"
+          --pre-note ./assets/release-sprint/pre-note.md
+          --post-note ./assets/release-sprint/post-note.md
+          --note-contributors innobead
+          --note-section-disable
+          --since-days 14
+          --exclude-labels "kind/test"
+          --exclude-labels "area/infra"
+          --exclude-labels "area/ci"
+          --exclude-labels "area/qa"
+          --exclude-labels "wontfix"
+          --exclude-labels "duplicated"
+          --exclude-labels "invalid"
+          --exclude-labels "release/task"
+          --pre-hook ./scripts/check-images-ready.sh
+          --pre-hook-args "longhornio/longhorn-manager:${{ steps.var.outputs.tag }}"
+          --pre-hook-args "longhornio/longhorn-engine:${{ steps.var.outputs.tag }}"
+          --pre-hook-args "longhornio/longhorn-instance-manager:${{ steps.var.outputs.tag }}"
+          --pre-hook-args "longhornio/longhorn-share-manager:${{ steps.var.outputs.tag }}"
+          --pre-hook-args "longhornio/backing-image-manager:${{ steps.var.outputs.tag }}"
+          --pre-hook-args "longhornio/longhorn-ui:${{ steps.var.outputs.tag }}"
+          --pre-hook-args "longhornio/longhorn-cli:${{ steps.var.outputs.tag }}"
```

Comment:
Need to remove the cli as it's only required for 1.7 and following versions.

Reply:
Updated. Thank you.